run_buyer_frontend.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Hmm, yeah I was deliberately making them with different names (renaming from the original ``run.sh``), but it might be better to have the same in each app.  I'll sort it now.